packages/core/src/sagas/sendPostBackToPostActivitySaga.js

```diff
@@ -1,19 +1,20 @@
 import {
   put,
-  take
+  takeEvery
 } from 'redux-saga/effects';
 
-import whileConnected from './effects/whileConnected';
-
 import { SEND_POST_BACK } from '../actions/sendPostBack';
 import postActivity from '../actions/postActivity';
+import whileConnected from './effects/whileConnected';
 
 export default function* () {
   yield whileConnected(function* () {
-    for (;;) {
-      const { payload: { value } } = yield take(SEND_POST_BACK);
-
-      if (value) {
+    yield takeEvery(
+      ({ payload, type }) => (
```

Comment:
inlining this takes away from the readability for me. can it be defined outside the function and given a good name?